### ISSUE TITLE
SMS notifications for field users [0.5+X]

### ISSUE THREAD

**Issue by MartinWahnschaffeSymeda** [hzi-braunschweig/SORMAS-Open#506]
Officers and informants need to receive an SMS notifications for certain trigger events. Thisway they can be also be informed in areas with no data connectivity.
To get the exact data they still have to synchronize data.

Depends on #292

Trigger events:

- [x] tasks to be started or due - following the same rules as notifications in app
- [x] weekly reports - also same rules as in app -> already done by creating a submit weekly report task

**Comment by hzi-braunschweig** [comment]
@MartinWahnschaffeSymeda @MateStrysewskeSym 
1. Task for  case notification, case investigation, contact follow up visits (all)
2. When lab sample of cases disposition from any state (confirmed, probable, suspected, not a case)
3. When a contact becomes symptomatic and changes to a case or a contact is lost to follow up
4. Tasks due when cases and contacts are left as not yet classified and unconfirmed contact respectively. how long before such a notification goes out? Not sure. maybe 48 hours
5. When rumor events are created and cases created from the rumor
6.  When an outcome is changed from one state to the other
7. When a case is moved from one  health facility to the other or moved from one LGA/state to the other
8. When general tasks are created on SORMAS. 
9. When lab sample of cases has been sent to a lab, the lab users should get a text message etc
10. When a task has been completed by an officer, the supervisors in charge should get a notification text that task has been completed. 
11. Any other sensible tasks that I may have left out

**Comment by MartinWahnschaffeSymeda** [comment]
Telephone call with Daniel:

- basically all points will automatically be handled by the existing process and/or generated tasks that will result in (SMS) notifications
- [ ] in addition we need an easy way for lab users to realize that new samples will arrive soon or an expected sample is not arriving (has likely not been sent) -> create seperate issue
- [ ] create an automatic task for supervisors to define the outcome of a case when it is still missing -> create seperate issue
- [ ] supervisors should get a notification when case investigation is done or contact visit is done
